sagemaker-pipelines/tabular/tuning-step/tuning_step_example.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        model_data=step_tuning.get_top_model_s3_uri(
</pre><p>can we add a sentence to describe <code>get_top_model_s3_uri </code> helper in the section intro? </p><p> </p><p>e.g.</p><p> </p><p>You use the&nbsp;<code>get_top_model_s3_uri</code>&nbsp;method of the&nbsp;<code>TuningStep</code>&nbsp;class to get the model artifact from one of the top performing model versions</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>